config/routes.rb

```diff
@@ -36,32 +40,47 @@
 
   # Simple Smart Answer pages
   get ":slug/y(/*responses)" => "simple_smart_answers#flow", :as => :smart_answer_flow
-  get ":slug", to: "simple_smart_answers#show", constraints: FormatRoutingConstraint.new('simple_smart_answer')
+  constraints FormatRoutingConstraint.new('simple_smart_answer') do
+    get ":slug", to: "simple_smart_answers#show"
+    get ":slug/:part", to: redirect('/%{slug}') # Support for simple smart answers that were once a format with parts
+  end
 
-  # Help pages
-  get "/help", to: "help#index"
-  get "/tour", to: "help#tour"
-  get "*slug", slug: %r{help/.+}, to: "help#show", constraints: FormatRoutingConstraint.new('help_page')
+  # Campaign pages.
+  get "/ukwelcomes", to: "campaign#uk_welcomes" # This is a hardcoded page in Frontend and does not have an associated artefact
+  constraints FormatRoutingConstraint.new('campaign') do
+    get ":slug", to: "campaign#show"
+    get ":slug/:part", to: redirect('/%{slug}') # Support for campaigns that were once a format with parts
```

Comment:
Thanks @sihugh, have removed the routing for those formats and updated the commit message